docs/cookbook/importing_settings_from_a_bundle.md

```diff
@@ -104,51 +95,37 @@ class AcmeTestExtension extends Extension implements PrependExtensionInterface
      */
     public function prepend( ContainerBuilder $container )
     {
-        // Loading our YAML file containing our template rules
+        // Loading your YAML file containing template rules
         $configFile = __DIR__ . '/../Resources/config/template_rules.yml';
         $config = Yaml::parse( file_get_contents( $configFile ) );
-        // We explicitly prepend loaded configuration for "ezpublish" namespace.
-        // So it will be placed under the "ezpublish" configuration key, like in ezpublish.yml.
+        // Explicitly prepend loaded configuration for "ezpublish" namespace.
+        // It will be placed under the "ezpublish" configuration key, like in ezplatform.yml.
         $container->prependExtensionConfig( 'ezpublish', $config );
         $container->addResource( new FileResource( $configFile ) );
     }
 }
 ```
 
+In `AcmeExampleBundle/Resources/config/template_rules.yml`:
+
 ``` yaml
-# AcmeTestBundle/Resources/config/template_rules.yml
-# We explicitly prepend config for "ezpublish" namespace in service container extension, 
+# You explicitly prepended config for "ezpublish" namespace in the service container extension, 
 # so no need to repeat it here
 system:
-    ezdemo_frontend_group:
-        ezpage:
-            layouts:
-                2ZonesLayout1:
-                    name: "2 zones (layout 1)"
-                    template: "AcmeTestBundle:zone:2zoneslayout1.html.twig"
-
+    site_group:
         content_view:
             full:
-                article_test:
-                    template: "AcmeTestBundle:full:article_test.html.twig"
+                article:
+                    template: "AcmeExampleBundle:full:article.html.twig"
                     match:
-                        Id\Location: 144
-                another_test:
-                    template: "::another_test.html.twig"
+                        Identifier\ContentType: [article]
+                special:
+                    template: "::special.html.twig"
                     match:
                         Id\Content: 142
-
-        block_view:
-            campaign:
-                template: "AcmeTestBundle:block:campaign.html.twig"
-                match:
-                    Type: "Campaign"
 ```
 
-!!! note "Regarding performance"
+!!! note "Performance"
 
-    Service container extensions are called only when the container is being compiled, so there is nothing to worry about regarding performance.
-
-!!! tip
-
-    Configuration loaded this way will be overridden by the main `ezplatform.yml` file.
+    Service container extensions are called only when the container is being compiled,
+    so performance should not be affected.
```

Comment:
I think we were more affirmative in the previous formulation. "The performance is not affected."